src/nidigital/metadata/enums.py

```diff
@@ -147,6 +165,14 @@
             }
         ]
     },
+    'HistoryRAMMaxSamplesToAcquirePerSite': {
```

Comment:
This didn't exist prior to its addition to grpc metadata. Why was it added?

Reply:
I think we should set `'codgen_method'`: 'no' for this, but it's not being added to the generated code, so it probably doesn't need to hold up this PR.

Reply:
I have a [PR](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/373719) ready to commit that will allow this. After that, I'll go apply it to all of those enums. But I'll make those changes separate from this PR.